pyproject.toml

```diff
@@ -91,6 +91,10 @@ build_dir = "jupytercad/labextension"
 
 [tool.jupyter-releaser.options]
 version-cmd = "python scripts/bump-version.py"
+python_packages = [
+    "jupyverse:fps-jupytercad",
+    ".:jupytercad:fps-jupytercad"
```

Comment:
Why do have 2 entries for `fps-jupytercad`?

Reply:
First line means that `fps-jupytercad` is in the `jupyverse` directory.
Second line means that `jupytercad` is in the root directory, and depends on `fps-jupytercad`.

Reply:
Yes, and `fps-jupytercad` manages versions through `hatch`, I don't understand why it doesn't work.
I tried https://github.com/QuantStack/jupytercad/pull/179/commits/2957b56717940a398d9df628b30104fcf0e66b11.